packages/provider-elasticsearch/src/example-types/metricGroups/tagsQueryGroupStats.js

```diff
@@ -1,7 +1,7 @@
 import { groupStats } from './groupStatUtils.js'
 import tagsQuery from '../filters/tagsQuery.js'
 
-let { buildResultQuery, filter } = tagsQuery
+let { buildResultQuery, filter } = tagsQuery(() => [])
```

Comment:
this invocation is wrong since tagsQuery takes an object - and with the maybeCall changes, you can just pass in nothing